jwt_proxy/config.py

```diff
@@ -6,6 +6,9 @@
 
 
 # $KEYCLOAK_FQDN/auth/realms/$REALM/protocol/openid-connect/certs
+OIDC_AUTHORIZE_URL = os.getenv("OIDC_AUTHORIZE_URL")
+OIDC_TOKEN_URI = os.getenv("OIDC_TOKEN_URI")
+OIDC_TOKEN_INTROSPECTION_URI = os.getenv("OIDC_TOKEN_INTROSPECTION_URI")
 JWKS_URL = os.getenv("JWKS_URL")
```

Comment:
Eventually, we should replace all of these individual configs with by parsing the OIDC well-known files that come with Keycloak

eg
https://keycloak.acc.dev.cosri.cirg.washington.edu/auth/realms/fEMR/.well-known/openid-configuration

Reply:
great idea - added TODO to code to capture this.